### PR TITLE
remove parameters from common-types networksecurityperimeter.json

### PR DESCRIPTION
**Pull request by TimLovellSmith**
because nobody can use them without linter warnings about needing to define a pattern (and a one-size-fits-all pattern is unknown.

Of course, nobody is using them yet.

### CODE REVIEW

**Review by mikeharder — APPROVED** [review]

**Comment by TimLovellSmith** [comment]
Hm, its saying I'm not authorized to merge this, I wonder if my write access expired.

**Comment by mikeharder** [comment]
> Hm, its saying I'm not authorized to merge this, I wonder if my write access expired.

You can re-apply for write access at https://aka.ms/azsdk/access

**Pull request closed without merging**
